### ISSUE TITLE
More informative errors / traceback?

### ISSUE THREAD

**Issue by msyriac** [CobayaSampler/cobaya#121]
There's probably an easy way to do this but I couldn't find it from looking at the docs. 

Say that my yaml file points to an external likelihood in a module in the same directory. I then insert a syntax error into the module that would prevent it from even being imported. I run the cobaya wrapper

```
$ cobaya-run act.yml --debug
..... LOTS OF MESSAGES....
 2020-10-27 12:51:16,296 [numba.core.ssa] ================================================================================
 2020-10-27 12:51:17,551 [tools] *ERROR* There was a problem when importing likelihood 'act_pylike.act100':
 2020-10-27 12:51:17,551 [input] *ERROR* Failed to get defaults for component or class 'act_pylike.act100' [No module named 'cobaya.likelihoods.act_pylike']
```

Ok, this tells me there was *some* error. But it doesn't tell me whether it was a silly missing parantheses, or some file the likelihood couldn't find or something else. Any source of error will give me the same message above. How/why does cobaya throw out the traceback for errors? Would it be possible to always print the full traceback?

I'm on the master branch and up-to-date, though I've seen this problem back in January too.

**Comment by cmbant** [comment]
If you know a way to get something helpful from errors inside _bootstrap._gcd_import then we could, but I don't see an immediate way to get anything helpful from it. Probably best to test inside the external likelihood first.

**Comment by msyriac** [comment]
I've submitted a fix in #134 that uses the `traceback` module. This solves the issue I was having, but I'm not sure if it will make things unnecessarily verbose in other situations.
